--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .gitignore

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/.gitignore b/.gitignore
index daf46f0..50d9c7a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -21,4 +21,5 @@ coverage/
 *.orig
 *.rej
 *.bk
-logs
\ No newline at end of file
+logs
+cbf_env/
\ No newline at end of file