--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   README.md
	modified:   logs/plots/comparison_results.pdf
	modified:   logs/plots/mean_episode_reward_log_summary.pdf

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/navigation_cbf/navigation_cbf_20260524_121158/
	logs/navigation_filter_only/navigation_filter_only_20260524_121421/
	logs/navigation_naive/navigation_naive_20260524_100904/
	logs/navigation_reward_only/navigation_reward_only_20260524_121327/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/README.md b/README.md
index f39b5ab..b3f0e01 100644
--- a/README.md
+++ b/README.md
@@ -73,6 +73,13 @@ python plot_tb_reward_log_steps.py \
     --soft-cbf logs/navigation_reward_only/.../events.out.tfevents... \
     --out-main logs/plots/mean_episode_reward.pdf
 ```
+
+Here are the specific files that were run on thorax GPU:
+
+``` bash
+python plot_tb_reward_log_steps.py     --naive logs/navigation_naive/navigation_naive_20260514_074524/events.out.tfevents.1778769928.thorax.238343.0     --cbf logs/navigation_cbf/navigation_cbf_20260514_084143/events.out.tfevents.1778773306.thorax.256096.0     --only-cbf logs/navigation_filter_only/navigation_filter_only_20260514_100151/events.out.tfevents.1778778114.thorax.275936.0     --soft-cbf logs/navigation_reward_only/navigation_reward_only_20260514_104520/events.out.tfevents.1778780723.thorax.291840.0     --out-main logs/plots/comparison_results.pdf
+```
+
 This generates PDF plots of the training rewards and obstacle collisions over time. The main plot is saved to `logs/plots/mean_episode_reward.pdf` and a summary bar chart to `logs/plots/mean_episode_reward_summary.pdf`.
 
 How to see the logs of the training process:
diff --git a/logs/plots/comparison_results.pdf b/logs/plots/comparison_results.pdf
index 7b69302..87562a9 100644
Binary files a/logs/plots/comparison_results.pdf and b/logs/plots/comparison_results.pdf differ
diff --git a/logs/plots/mean_episode_reward_log_summary.pdf b/logs/plots/mean_episode_reward_log_summary.pdf
index 7fe0f45..b847fff 100644
Binary files a/logs/plots/mean_episode_reward_log_summary.pdf and b/logs/plots/mean_episode_reward_log_summary.pdf differ